Packs/CommonScripts/Scripts/BlockExternalIp/BlockExternalIp.py

```diff
@@ -886,7 +866,8 @@ def main():  # pragma: no cover
                             'rule_name': rule_name,
                             'auto_commit': auto_commit,
                         }
-                        results.append(prisma_sase_block_ip(brand_args))
+                        prisma_sase = PrismaSase(brand_args, [])
```

Comment:
Is there a situation where we pass the responses to the class init? If so, why do we get it as an arg? lets init it in the init of the class.